libs/utils/utils.js

```diff
@@ -346,6 +346,47 @@ export async function loadTemplate() {
   await Promise.all([styleLoaded, scriptLoaded]);
 }
 
+export async function decorateLinksToButtons(links) {
```

Comment:
I think it was fine to have in a separate file. We need utils to get smaller, not larger.

Reply:
I've moved it :)